Update references to published Paper; modified citation style

nichtmonti opened the pull request:
The Master Thesis by Clermont has since been published at Asiacrypt2024 and the Master Thesis by Wilson has been published at ASIACCS. The references have been updated to point to the conference version and to acknowledge the co-authors.

Furthermore, the citation style of [Clermont], [Wilson], [Frymann2020] and [Frymann2023] has been updated to match the convention used by Cryptobib to allow for an easy identification of the venue where it was published and the authors involved in the project.

nichtmonti committed:
Update Reference from Master Thesis to published Paper; modified citation style

nichtmonti committed:
fix citation for paper with three authors

nichtmonti committed:
Reference ASIACCS Paper instead of Wilson thesis

emlun commented:
Thank you! I'll have to confirm that the new sources still match up with what we're referencing, but on the surface these look like reasonable changes.

nichtmonti committed:
Merge branch 'main' into resolve-conflicts

emlun committed:
Delete extraneous comma

emlun committed:
Restore Wilson2023 reference for BL construction

The revision in [ASIACCS:SteWil24] replaces the "key blinding scheme"
component with a "key-blinding signature scheme", which is not
one-for-one compatible with the ARKG construction in the current
revision of this specification.

emlun committed:
Fix undefined references

emlun reviewed (approved):
Thanks! I reverted the references to [Wilson2023] for the `BL` component, since [ASIACCS:SteWil24] replaced the "key blinding scheme" component with a "key-blinding signature scheme" component which is not one-for-one compatible with the current ARKG construction in this document.

I think we can update the construction here as an editorial revision to use "key-blinding signature scheme" instead without affecting implementations, but we'll do that separately if so. It would also narrow the scope from key generation for any purpose to only signature key generation, which I'm not sure we actually want.

ve7jtb reviewed (commented):
LGTM
